lib/rouge/lexers/php.rb

```diff
@@ -28,6 +28,10 @@ def initialize(*)
         @disabledmodules = list_option(:disabledmodules)
       end
 
+      def start_inline?
```

Comment:
Should [this line](https://github.com/rouge-ruby/rouge/blob/0b57e9060ec25a76cf788251cd7b2a0acc6d7f15/lib/rouge/lexers/php.rb#L58) be updated to use this method?

Reply:
I generally try to keep method calls out of lexing as much as possible, since it can be hot code. `start` shouldn't be an issue though

Reply:
Oh wait, no, it can be `:guess` which is special behaviour. The query method only returns a boolean.